koku/api/settings/test/tag_mappings/test_view.py

```diff
@@ -0,0 +1,288 @@
+import json
+import random
+
+from django.urls import reverse
+from django_tenants.utils import tenant_context
+from faker import Faker
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.test import APIClient
+
+from api.iam.test.iam_test_case import IamTestCase
+from api.settings.tag_mappings.query_handler import format_tag_mapping_relationship
+from api.settings.tag_mappings.view import SettingsTagMappingFilter
+from reporting.provider.all.models import EnabledTagKeys
+from reporting.provider.all.models import TagMapping
+
+
+class SettingsTagMappingViewTestCase(IamTestCase):
+    def setUp(self):
+        """Set up the tests."""
+        super().setUp()
+        self.client = APIClient()
+
+        # Create some sample data
+        cloud_providers = ["AWS", "Azure", "GCP", "OCP"]
+        fake = Faker()
+
+        for _ in range(30):
+            tag_key = EnabledTagKeys(key=fake.uuid4(), enabled=True, provider_type=random.choice(cloud_providers))
+            with tenant_context(self.tenant):
+                tag_key.save()
```

Comment:
This bit of code runs for every single test case in this class. Do we even need this data? The model bakery builds a known set of enabled tags [here](https://github.com/project-koku/koku/blob/de9b60bf7874e1356df187bbfaa0e084cad8cd61/koku/api/report/test/util/model_bakery_loader.py#L85) which is run with the koku-test-runner [here](https://github.com/project-koku/koku/blob/85284b1e8489cd573d4049db0e0940091868dc68/koku/koku/koku_test_runner.py#L1)

Reply:
Switched to just using the preloaded bakery stuff.